docs/ja/ai-integration/tools-json.md

```
@@ -1,55 +1,49 @@
+| エンドポイント | 形式 | 説明 |
 |---------------|------|------|
-| `GET /llms.txt` | Markdown (llms.txt) | API documentation for LLMs |
-| `GET /tools.json` | JSON | Claude Tool Use / OpenAI Function Calling compatible schema |
-| `GET /.well-known/ai-plugin.json` | JSON | AI plugin manifest |
-| `GET /openapi.json` | JSON | OpenAPI 3.0 specification |
-| `GET /api.json` | JSON | API reference |
+| `GET /llms.txt` | Markdown (llms.txt) | LLM 向け API ドキュメント |
+| `GET /tools.json` | JSON | Claude Tool Use / OpenAI Function Calling 互換スキーマ |
+| `GET /.well-known/ai-plugin.json` | JSON | AI プラグインマニフェスト |
+| `GET /openapi.json` | JSON | OpenAPI 3.0 仕様 |
+| `GET /api.json` | JSON | API リファレンス |## Tool Use Schema (`/tools.json`)
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)